src/de/willuhn/jameica/hbci/gui/controller/EinnahmeAusgabeControl.java

```diff
@@ -321,7 +324,13 @@ public void format(TreeItem item)
         calendar.setTimeInMillis(calendar.getTime().getTime()-1);
         Date nodeTo = DateUtil.startOfDay(calendar.getTime());
         
-        List<EinnahmeAusgabe> werte = this.getWerte(nodeFrom, nodeTo);
+        List<EinnahmeAusgabe> werte = this.getWerte(nodeFrom, nodeTo, vorZeitraumsWerte);
+        for (EinnahmeAusgabe einnahmeAusgabe : werte)
+        {
+          if(einnahmeAusgabe.getKonto()!=null) {
+            vorZeitraumsWerte.put(einnahmeAusgabe.getKonto().getID(), einnahmeAusgabe);
+          }
+        }
```

Comment:
nachdem die Werte für den Zeitraum ermittelt sind, packe ich sie in die Map für die nächste Iteration